src/components/languages/LanguageSelect.js

```diff
@@ -41,14 +42,30 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 function LanguageSelect({ language, onLanguage }) {
+  const appContext = useContext(AppContext);
   const classes = useStyles();
 
   const handleChange = (object) => {
     const languageId = object.value;
```

Comment:
@mandolyte , at first it wasn't working
tonight I debugged it to see `object.value` isn't defined.  Instead, its `object.langId`